lib/ui/proposals/creation/components/proposal_content_view.dart

```diff
@@ -27,6 +28,13 @@ class _ProposalContentViewState extends State<ProposalContentView> {
   void initState() {
     super.initState();
 
+    _titleController.text = context.read<ProposalCreationBloc>().state.proposal!.title ?? '';
```

Comment:
IMO any time there's a ! that's a bug. 

Generally - there may be exceptions. 

Decide if proposal is always defined - if it is always defined, make it non-optional, meaning, it will always be defined in the codebase and at static compile time. 

If it is nullable - then use ? and deal with situations where it's null. 

The entire point of the nullable / non-nullable architecture is to force developers to think whether something can be null. This prevents the most prevalent bug out there, the null pointer exception. Using "!" gets around the architecture and should be done only as an exception.

Reply:
So in this case
Preferred solution is proposal is not nullable

Second best is use ? you're already handling this anyway in this code